test/system/600-completion.bats

```diff
@@ -231,6 +238,23 @@ function _check_completion_end() {
     is "${lines[-1]}" "Completion ended with directive: ShellCompDirective$1" "Completion has wrong ShellCompDirective set"
 }
 
+# check that there no suggestions in the output
+# we could only check stdout and not stderr but this is not possible with bats
+# by default we always have to extra lines at the end
+# and then we could have other extra lines for debugging, they will always start with [Debug]
```

Comment:
This entire comment is a case where it would be really nice to have a real-world example, like
```
# for example, podman foo bar returns a bunch of [Debug] lines followed by (whatever)
```